src/components/Card.tsx

```diff
@@ -41,14 +17,33 @@ const Card: React.FC<IProps> = ({
   children,
   className,
 }) => (
-  <StyledCard className={className}>
-    <TopContent>
-      {emoji && <StyledEmoji size={3} text={emoji} />}
-      {title && <Title>{title}</Title>}
-      {description && <Description>{description}</Description>}
-    </TopContent>
+  <Flex
+    direction="column"
+    bg="ednBackground"
+    borderRadius="2px"
+    border="1px"
+    borderStyle="solid"
+    borderColor="lightBorder"
+    p={6}
+    className={className}
```

Comment:
Same as in the other PR, I think it is a good moment to deprecated the usage of `className` in our components.

Reply:
As commented in a different PR, we are going to keep these `className`s until we remove the remaining styled components who use this prop.